### PR TITLE
v1.0.0 🍾

### LINKED ISSUE #64: Custom data attribute not working or misunderstand its usage?
Reproducible steps:

1. Input data-ved or ved in settings of extension
Screen Shot 2019-09-12 at 11 50 38 AM

2. Go to "google.com"
3. Press "Google Search" button
4. Received code
```
const puppeteer = require('puppeteer');
(async () => {
  const browser = await puppeteer.launch()
  const page = await browser.newPage()
  
  await page.goto('https://www.google.com/')
  
  await page.setViewport({ width: 983, height: 685 })
  
  await page.waitForSelector('div > .A8SBwf > .FPdoLc > center > .gNO89b')
  await page.click('div > .A8SBwf > .FPdoLc > center > .gNO89b')
  
  await browser.close()
})()
```
5. Expected
```
const puppeteer = require('puppeteer');
(async () => {
  const browser = await puppeteer.launch()
  const page = await browser.newPage()
  
  await page.goto('https://www.google.com/')
  
  await page.setViewport({ width: 983, height: 685 })
  
  await page.waitForSelector('[data-ved="0ahUKEwjG8ZGz-MvkAhXRqZ4KHb3SClEQ4dUDCAc"]')
  await page.click('[data-ved="0ahUKEwjG8ZGz-MvkAhXRqZ4KHb3SClEQ4dUDCAc"]')
  
  await browser.close()
})()
```

### PR DESCRIPTION
**Pull request by ianaya89**
# v1.0.0 🍾 

## Description

image

### Added
- New visual identity by [@nucro](https://twitter.com/nucro).
- In page overlay to handle recording and take screenshots
- Visual feedback when taking screenshots
- New code structure organized in modules and services
- Dark mode support
- Migrate to Vue 3 and dependencies update
- Migrate CSS to Tailwind (except for Overlay components)
- Selector preview while recording
- Restart button while recording
- Allow run scripts directly on Checkly 🦝
- First draft of Vuex shared store

### Changed
- Make Playwright default tab
- Use non-async wrap as default
- Full-page screenshots use `fullPage` property
- Replace clipped screenshots with element screenshots
- Improve selector generation giving relevance to `ID` and `data-attributes`
- General bug fixing
- Improve code reusability and events management

### Removed
- Screenshots context menu
- Events recording list

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. 

## Checklist:

- [x] My code follows the style guidelines of this project. `npm run lint` passes with no errors.
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works (no yet)
- [ ] New and existing unit tests pass locally with my changes. `npm run test` passes with no errors. (no yet)

> Resolves #64

### CODE REVIEW

File: src/background/index.js [hunk 1]
```diff
@@ -1,237 +1,305 @@
...
-       */
+    this._boundedMessageHandler = this.handleMessage.bind(this)
+    this._boundedNavigationHandler = this.handleNavigation.bind(this)
+    this._boundedWaitHandler = () => badge.wait()
 
-      chrome.contextMenus.removeAll()
+    this.overlayHandler = this.handleOverlayMessage.bind(this)
 
-      // add the parent and its children
+    // chrome.contextMenus.create({
```
Comment (maxigimenez) [inline src/background/index.js:53]: remove?

File: src/background/index.js [hunk 2]
```diff
@@ -1,237 +1,305 @@
...
-  stop () {
-    console.debug('stop recording')
+  stop() {
     this._badgeState = this._recording.length > 0 ? '1' : ''
 
     chrome.runtime.onMessage.removeListener(this._boundedMessageHandler)
     chrome.webNavigation.onCompleted.removeListener(this._boundedNavigationHandler)
     chrome.webNavigation.onBeforeNavigate.removeListener(this._boundedWaitHandler)
-    chrome.contextMenus.onClicked.removeListener(this._boundedMenuHandler)
+    // chrome.contextMenus.onClicked.removeListener(this._boundedMenuHandler)
```
Comment (maxigimenez) [inline src/background/index.js:84]: this should be commented out?

File: src/components/Footer.vue [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    <a href="https://checklyhq.com" target="_blank">
+      <img src="/images/checkly-logo.svg" alt="Checkly logo" class="w-24" />
+    </a>
+    <span class="text-gray-darkish">Version {{ version }}</span>
+  </div>
+</template>
+
+<script>
+import { ref } from 'vue'
+import { version } from '../../package.json'
```
Comment (maxigimenez) [inline src/components/Footer.vue:12]: nit: `@/package.json`?

File: src/components/Header.vue [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+      </button>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {}
+</script>
+
+<style></style>
```
Comment (maxigimenez) [inline src/components/Header.vue:24]: remove

File: src/components/Header.vue [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+        <img src="@/assets/icons/question.svg" alt="help" class="w-4" />
+      </button>
+      <button @click="$emit('options')" class="ml-2">
+        <img src="@/assets/icons/gear.svg" alt="settings" class="w-4" />
+      </button>
+    </div>
+  </div>
+</template>
+
+<script>
```
Comment (maxigimenez) [inline src/components/Header.vue:20]: this can be removed?

**Review by maxigimenez — CHANGES_REQUESTED** [review]
Few small nits and questions.

Great work!!! 👏

File: src/components/Footer.vue (hunk 3 above)
Comment (ianaya89) [reply src/components/Footer.vue:12]: `@` aliases `src` folder, so package is out of scope

File: src/background/index.js (hunk 2 above)
Comment (ianaya89) [reply src/background/index.js:84]: Is a new feature I am working on but not finished yet

**Comment by ianaya89** [comment]
@maxigimenez thank you :D

**Pull request merged**